hack/lib/testselect.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
I'm getting
```
./hack/../../hack/lib/testselect.bash: line 13: testselect: command not found
15:33:37.581 ERROR:   🚨 Error (code: 127) occurred at ./hack/../../hack/lib/testselect.bash:13, with command: testselect --testsuites="${rootdir}/test/testsuites.yaml" --clonerefs="${ARTIFACT_DIR}/clonerefs.json" --output="${ARTIFACT_DIR}/tests.txt"
make[1]: *** [generated-files] Error 127
```
on https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-knative_eventing/157/pull-ci-openshift-knative-eventing-release-v1.8-410-test-conformance-aws-ocp-410/1636020994751598592

we're calling `make generated-files` in eventing core when running in prow

Reply:
I guess the check on `[[ -n "${ARTIFACT_DIR:-}" ]]` is not enough for all cases or maybe we can install testselect regardless?

Reply:
Hmm, the command `GO111MODULE=off go get github.com/openshift-knative/hack/cmd/testselect` probably doesn't work there because all of this code (installing and calling testselect) is behind the ARTIFACT_DIR check.
Looking into it.

Reply:
It's caused by this line: https://github.com/openshift-knative/eventing/blob/release-v1.8/openshift/e2e-common.sh#L86
After that, `go get` doesn't work.

Reply:
@pierDipi The fix is here: https://github.com/openshift-knative/serverless-operator/pull/2003

Reply:
Thanks!